SCClassLibrary/Common/Audio/SynthDef.sc

```diff
@@ -573,7 +573,7 @@ SynthDef {
 	send { arg server, completionMsg;
 		var servers = (server ?? { Server.allRunningServers }).asArray;
```

Comment:
The new nomenclature that distinguishes between `hasBooted` and `serverRunning` is very good: you can read in the code that `allRunningServers` can't be quite right here.